install/kubernetes/helm/subcharts/certmanager/templates/issuer.yaml

```diff
@@ -12,7 +12,9 @@ metadata:
 spec:
   acme:
     server: https://acme-staging-v02.api.letsencrypt.org/directory
+{{- if .Values.email }}
```

Comment:
Isn't this required ? I don't think it works without an email

Reply:
Issue is reproducible using `helm template ./install/kubernetes/helm/subcharts/certmanager/ --set global.image=certmanager`. 

Output YAML:

```yaml
acme:
  server: https://acme-staging-v02.api.letsencrypt.org/directory
  email:
```
corresponding JSON is

```json
"acme": {
  "server": "https://acme-v02.api.letsencrypt.org/directory",
  "email": null
}
```

Reply:
So after your change generated json would be:
```json
"acme": {
  "server": "https://acme-v02.api.letsencrypt.org/directory",
}
```
Will it work with no email field at all? I guess that's what @costinm commented